dashboard/src2/components/site/SitePerformance.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+	<div class="space-y-4">
+		<AlertBanner
+			title="All metrics presented are from the last 24 hours."
+			type="info"
+		/>
+		<div>
+			<ObjectList :options="slowQueriesData" />
+		</div>
+	</div>
+</template>
+
+<script lang="jsx">
```

Comment:
```suggestion
<script>
```

Since we aren't using any jsx here